cmd/loadtest.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/topfreegames/khan/loadtest"
+	"github.com/topfreegames/khan/log"
+	"github.com/uber-go/zap"
+)
+
+var sharedClansFile string
+
+var loadtestCmd = &cobra.Command{
+	Use:   "loadtest",
+	Short: "runs a load test against a remote Khan API",
+	Long: `Runs a load test against a remote Khan API with the specified arguments.
+You can use environment variables to override configuration keys.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		logger := zap.New(zap.NewJSONEncoder(), zap.InfoLevel)
+
+		l := logger.With(
+			zap.String("source", "loadtestCmd"),
+			zap.String("operation", "Run"),
+		)
+
+		app := loadtest.GetApp(ConfigFile, sharedClansFile, logger)
+		if err := app.Run(); err != nil {
+			log.P(l, "Application exited with error.", func(cm log.CM) {
+				cm.Write(zap.String("error", err.Error()))
+			})
+		} else {
+			log.I(l, "Application exited without errors.")
+		}
+	},
+}
+
+func init() {
+	RootCmd.AddCommand(loadtestCmd)
+
+	loadtestCmd.Flags().StringVar(
+		&sharedClansFile,
+		"clans",
+		"./config/loadTestSharedClans.yaml",
```

Comment:
why a file instead of a env var with a list?

Reply:
My plan is to mount a configMap! So I define the configMap once and just mount it in the pods.

Reply:
I mean, mount it once in the deployment/replicaset.

Reply:
Why do you think this is better than the env var + sensible default values approach?

Reply:
Because it may be a very long list of clan public IDs, I'm not sure it's a good idea to have env vars that long... Also I don't get what you mean about "sensible default values" in this case.

Reply:
Sorry, I confused the threads. Sensible default values was meant for the probabilities.

Reply:
I agree the list can be long, we can move with the config map approach and may change this in the future if it doesn't work for us.